src/test/java/sorinfoca/Tests/Tema8/pages/RefreshPage.java

```diff
@@ -0,0 +1,30 @@
+package sorinfoca.Tests.Tema8.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+import java.time.Instant;
+
+public class RefreshPage {
+
+    private WebDriver driver;
+
+    public RefreshPage(WebDriver driver) {
+        this.driver = driver;
+    }
+
+    public void get() {
```

Comment:
Please rename to goToPage() or something that suggests this action of navigating to url. In this way it will be easier to understand what this method do and not confusing with a getter method. Thanks!